doc/manuals/admin/upgrading_crossing_1-3-0.md

```diff
@@ -0,0 +1,90 @@
+# Upgrading to 1.3.0 and beyond from a pre-1.3.0 version
+
+This procedure has to be done always as long as your database contains
+at least one entity document.
+
+-   Stop contextBroker
+-   Remove previous contextBroker version
+
+        yum remove contextBroker
+
+-   [Take a backup of your
+    DBs](database_admin.md#backup) (this is just a
+    safety measure in case any problem occurs, e.g some script gets
+    interrupted before finished and your database data ends in an
+    incoherent state)
+-   Download the following script:
+    -   [mdsvector2mdsobject.py](https://github.com/telefonicaid/fiware-orion/blob/1.3.0/scripts/managedb/mdsvector2mdsobject.py)
+-   Install pymongo (it is a script dependency) in case you don't have
+    it previously installed
+
+        pip-python install pymongo
+
+-   Apply the mdsvector2mdsobject.py to your DBs, using the
+    following (where 'db' is the database name). Note that if you are
```

Comment:
I agree that is better as you said, but not sure how to Markdown/RTD deal with extra text at the same indentation level of a bullet but that is not a bullet itself.